framework/packages/abstract-client/src/client.rs

```diff
@@ -174,29 +174,37 @@ impl<Chain: CwEnv> AbstractClient<Chain> {
         Ok(builder)
     }
 
-    /// Fetches an existing Abstract [`Account`] from chain
-    pub fn fetch_account(&self, namespace: Namespace) -> AbstractClientResult<Account<Chain>> {
-        Account::maybe_from_namespace(&self.abstr, namespace.clone())?.ok_or(
-            AbstractClientError::NamespaceNotClaimed {
-                namespace: namespace.to_string(),
-            },
-        )
+    /// Fetch an [`Account`] from a given source.
+    ///
+    /// This method is used to retrieve an account from a given source. It will **not** create a new account if the source is invalid.
+    ///
+    /// Sources that can be used are:
+    /// - [`Namespace`]: Will retrieve the account from the namespace if it is already claimed.
+    /// - [`AccountId`]: Will retrieve the account from the account id.
+    /// - App [`Addr`]: Will retrieve the account from an app that is installed on it.
+    pub fn fetch_account<T: Into<AccountSource>>(
```

Comment:
this can be removed actually, it doesn't make sense to have that

Reply:
> this can be removed actually, it doesn't make sense to have that

We should remove this or `account_from`, what would you prefer to remove? @CyberHoward

Reply:
`account_from` could be mistaken for account creation where fetch_account is clear in that sens

Reply:
Lovely